src/AccessibilityInsights.SharedUxTests/Settings/AppLayoutTests.cs

```diff
@@ -96,8 +96,8 @@ public void AppLayoutTest()
         [TestMethod()]
         public void RemoveConfigurationTest()
         {
-            string path = Path.Combine(DirectoryManagement.sUserDataFolderPath, TestLayoutFile);
-            foreach (string file in System.IO.Directory.EnumerateFiles(DirectoryManagement.sUserDataFolderPath))
+            string path = Path.Combine(provider.UserDataFolderPath, TestLayoutFile);
+            foreach (string file in System.IO.Directory.EnumerateFiles(provider.UserDataFolderPath))
```

Comment:
>System.IO. [](start = 36, length = 10)

Line 7 specifies System.IO, but we're repeating it several times in this class. If it's not needed, can we do some opportunistic cleanup?